tests/e2e/specs/orderList_spec.js

```diff
@@ -6,67 +6,83 @@ describe('my orders', () => {
     password: 'p@ssword',
   };
 
-  const orderDraft1 = {
-    orderNumber: '1234',
-    paymentState: 'Pending',
-    shipmentState: 'Shipped',
-  };
+  describe('shows my orders', () => {
```

Comment:
Nice that you came up with the idea of `describe`, but it makes more sense when the setup of a group of tests differs. In this case though you only have one test on each "group" and they are both having the same setup:
```
cy.visit('/');
cy.changeLanguage('Deutsch');
cy.login(customer);
```
So the `describe` just makes the test longer without benefits. I would remove it in this case. WDYT?

Reply:
I noticed that you are using `before`. Have you tried with `beforeEach`?